etc/conf/ccTelemetry.yaml

```diff
@@ -53,6 +53,7 @@ end_collection_event:
 imageParam_event:
     device: CCCamera
     topic: imageReadoutParameters
+    array_keys: ccdLocation
```

Comment:
and this...

Reply:
Does Tony need to review this as well?

Reply:
Same. No, Tony doesn't need to review this.